MvvmCross.Android.Support/Design/MvxBottomNavigationView.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Windows.Input;
+using Android.Content;
+using Android.Runtime;
+using Android.Widget;
+using Android.Util;
+using Android.Views;
+using Android.Support.Design.Widget;
+
+namespace MvvmCross.Droid.Support.Design
+{
+    [Register("mvvmcross.droid.support.design.MvxBottomNavigationView")]
+    public class MvxBottomNavigationView : BottomNavigationView, BottomNavigationView.IOnNavigationItemSelectedListener
+    {
+        public List<MvxMenuItem> ItemsSource = new List<MvxMenuItem>();
+
+        public MvxBottomNavigationView(Context context) : base(context)
+        {
+            this.SetOnNavigationItemSelectedListener(this);
+        }
+
+        public MvxBottomNavigationView(Context context, IAttributeSet attrs) : base(context, attrs)
+        {
+            this.SetOnNavigationItemSelectedListener(this);
+        }
+
+        public MvxBottomNavigationView(Context context, IAttributeSet attrs, int defStyleAttr) : base(context, attrs, defStyleAttr)
+        {
+            this.SetOnNavigationItemSelectedListener(this);
+        }
+
+        protected MvxBottomNavigationView(IntPtr javaReference, JniHandleOwnership transfer) : base(javaReference, transfer)
+        {
+            this.SetOnNavigationItemSelectedListener(this);
+        }
+
+        public void AddItem(IMenuItem item, Type viewModel)
+        {
+            ItemsSource.Add(new MvxMenuItem(item, viewModel));
+
+            // The first item is autoselected
+            if (ItemsSource.Count == 1)
+            {
+                OnNavigationItemSelected(item);
```

Comment:
Maybe we need to make a bool that defaults to true if the user wants to auto open the first tab?